dev/reference/performance/statistics.md

```diff
@@ -15,24 +15,81 @@ Based on the statistics, the TiDB optimizer chooses the most efficient query exe
 
 ### Manual collection
 
-You can run the `ANALYZE` statement to collect statistics. Note that `ANALYZE TABLE` in TiDB takes considerably longer than in MySQL/InnoDB. In InnoDB, only a small number of pages are sampled, while in TiDB a comprehensive set of statistics is completely rebuilt. Scripts that were written for MySQL may naively expect `ANALYZE TABLE` will be a short-lived operation.
+You can run the `ANALYZE` statement to collect statistics.
 
-Syntax:
+#### Full collection
+
+> **Note:**
+>
+> `ANALYZE TABLE` in TiDB takes considerably longer than in MySQL/InnoDB. In InnoDB, only a small number of pages are sampled, while in TiDB a comprehensive set of statistics is completely rebuilt. Scripts that were written for MySQL may naively expect `ANALYZE TABLE` will be a short-lived operation.
+>
+> For quicker analysis, you can set `tidb_enable_fast_analyze` to `1` to turn on Quick Analysis. The default value for this parameter is `0`.
+>
+> When Quick Analysis is turned on, TiDB randomly samples approximately 10,000 rows of data to build statistics. Therefore, in the case of uneven data distribution or relatively small amount of data, the accuracy of statistical information will be relatively poor. It may lead to poor execution plans, such as choosing the wrong index. If the execution time of the normal `ANALYZE` statement is acceptable, it is recommended to turn off Quick Analysis.
```

Comment:
```suggestion
> After Quick Analysis is enabled, TiDB randomly samples approximately 10,000 rows of data to build statistics. Therefore, in the case of uneven data distribution or a relatively small amount of data, the accuracy of statistical information is relatively poor. It might lead to poor execution plans, such as choosing the wrong index. If the execution time of the `ANALYZE` statement is acceptable, it is recommended to disable the Quick Analysis feature.
```